js-api/src/chem.ts

```diff
@@ -582,6 +581,12 @@ export namespace chem {
       return imageHost;
     }
 
+    disableSketcher(message: string) {
```

Comment:
I don't like this. This is breaking the sketcher and there is no way to enable it. APIs should not be done like that. Also, a method like that should be documented. 

I suggest something like `setEnabled(error: string | null)`. And it shouldn't remove anything, just put a div on top (and then remove if needed)

Reply:
As we discussed yesterday, I will implement disabling outside chem.ts bot to break API